Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -1474,7 +1481,7 @@ private void OnValidate()
                 return;
             }
 
-            RegisterInstance(this);
+            RegisterInstance(Instance);
```

Comment:
I think this should be left as-is. The `Instance` getter also calls `RegisterInstance`, so we'd be calling it on the same instance twice here:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/d50838d18f0f8445357a66a974c0fc8fe11cb14e/Assets/MRTK/Core/Services/MixedRealityToolkit.cs#L554

So either left as-is or made only a reference to the `Instance` getter, like we do with 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/d50838d18f0f8445357a66a974c0fc8fe11cb14e/Assets/MRTK/Core/Services/MixedRealityToolkit.cs#L485-L486

I think I prefer as-is, since then we'll be explicitly registering the instance of the script that's actually in the scene and getting its `OnValidate` called, instead of searching the scene like the getter does.

Were you seeing cases where the existing code wasn't working as expected?